Content.Server/Delivery/DeliverySystem.Spawning.cs

```diff
@@ -0,0 +1,126 @@
+using Content.Server.Power.Components;
+using Content.Server.StationRecords;
+using Content.Shared.Delivery;
+using Content.Shared.EntityTable;
+using Robust.Shared.Random;
+using Robust.Shared.Timing;
+
+namespace Content.Server.Delivery;
+
+/// <summary>
+/// System for managing deliveries spawned by the mail teleporter.
+/// This covers for spawning deliveries.
+/// </summary>
+public sealed partial class DeliverySystem
+{
+    [Dependency] private readonly IGameTiming _timing = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] private readonly EntityTableSystem _entityTable = default!;
+
+    private void InitializeSpawning()
+    {
+        SubscribeLocalEvent<CargoDeliveryDataComponent, MapInitEvent>(OnDataMapInit);
+    }
+
+    private void OnDataMapInit(Entity<CargoDeliveryDataComponent> ent, ref MapInitEvent args)
+    {
+        ent.Comp.NextDelivery = _timing.CurTime + ent.Comp.MinDeliveryCooldown; // We want an early wave of mail so cargo doesn't have to wait
+    }
+
+    private void SpawnDelivery(Entity<DeliverySpawnerComponent?> ent, int amount)
+    {
+        if (!Resolve(ent.Owner, ref ent.Comp))
+            return;
+
+        var coords = Transform(ent).Coordinates;
+
+        if (ent.Comp.SpawnSound != null)
+            _audio.PlayPvs(ent.Comp.SpawnSound, ent.Owner);
```

Comment:
```suggestion
        _audio.PlayPvs(ent.Comp.SpawnSound, ent.Owner);
```
the function already checks this